newIDE/app/src/Utils/UseCreateProject.js

```diff
@@ -141,6 +138,9 @@ const useCreateProject = ({
 
         const oldProjectId = currentProject.getProjectUuid();
         initialiseProjectProperties(currentProject, newProjectSetup);
+        if (newProjectSource.templateSlug) {
```

Comment:
why moving this?

Reply:
I moved the templateSlug from the "new project setup" to the "new project source" (because arguably it's more an information coming from the source rather than the setup - the user has no choice to make about this), which allowed to generalize this to private templates and tutorial ids.